k8s/namespaces/family-public-law/fpl-case-service/demo.yaml

```diff
@@ -4,13 +4,13 @@ metadata:
   name: fpl-case-service
   annotations:
     fluxcd.io/automated: "true"
-    fluxcd.io/tag.java: glob:pr-3241-*
+    fluxcd.io/tag.java: glob:pr-3421-*
     hmcts.github.com/prod-automated: disabled
 spec:
   values:
     java:
       ingressHost: fpl-case-service-demo.service.core-compute-demo.internal
-      image: hmctspublic.azurecr.io/fpl/case-service:pr-3241-90ef0fe-20220316120714
+      image: hmctspublic.azurecr.io/fpl/case-service:pr-3421-8475484-20220318120910
```

Comment:
This requires an image policy setup for this - rather than manual change

https://github.com/hmcts/cnp-flux-config/blob/master/apps/family-public-law/fpl-case-service/demo-image-policy.yaml

Reply:
Moved change to use demo-image-policy